Resources/public/css/theme/views/content-creation.css

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-contentcreationwidgetview {
+    text-align: left;
+}
+
+.ez-view-contentcreationwidgetview .on-page-two {
+    transform: translateX(-100%);
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header {
+    border-bottom: 1px #000 solid;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__list {
+    list-style: none;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__btn--change-content-type {
+    background: #fff;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__btn--change-content-type:before,
+.ez-view-contentcreationwidgetview .cof-content-creation__btn--change-content-type:after {
+    content: "";
+    width: 1px;
+    height: 1rem;
+    background: #aaa;
+    position: absolute;
+    top: 50%;
+    right: 1rem;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__btn--change-content-type:before {
+    transform: translate(-50%, -50%) rotate(45deg);
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__btn--change-content-type:after {
+    transform: translate(-50%, -50%) rotate(-45deg);
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__text {
+    line-height: 1.5rem;
+    color: #aaa;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__text:before {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 1.5rem;
+    height: 1.5rem;
+    background: #aaa;
+    color: #fff;
+    border-radius: 50%;
+    line-height: 1.5rem;
+    text-align: center;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__text:first-child:before {
+    content: '1';
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__text:nth-child(2):before {
+    content: '2';
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__text--highlighted {
+    color: #000;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__header__text--highlighted:before {
+    background: #3FB499;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__page-one,
+.ez-view-contentcreationwidgetview .cof-content-creation__page-two {
+    transition: transform .3s cubic-bezier(.25,.8,.25,1);
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__content-type-selector {
+    border-radius: 5px;
+    background: #fdfdfd;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__content-type-selector--is-loading:before {
+    content: "";
+    height: 100%;
+    width: 100%;
+    background: #fdfdfd;
+    position: absolute;
+    top: 0;
+    left: 0;
+    border-radius: 5px;
+}
+
+.ez-view-contentcreationwidgetview .cof-content-creation__content-type-selector--is-loading:after {
+    display: block;
```

Comment:
It can be removed. Whenever you're using `position: absolute;` it becomes `display: block` by default.